A/Agama/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+using BinaryBuilder, Pkg
+
+name = "Agama"
+repo = "https://github.com/GalacticDynamics-Oxford/Agama.git"
+
+version = v"1.0"
+sources = [
+    GitSource(repo, "f392b9d6e59269d451b37b3ada2508a44bd3a80c"),
+]
+
+platforms = [
+    Platform("aarch64", "macos"),
+    Platform("x86_64", "linux")
+]
+
+products = [
+    LibraryProduct("agama", :libagama),
+]
+
+dependencies = [
+    Dependency("GSL_jll"),
+    Dependency("LLVMOpenMP_jll"),
+    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/Agama
+
+# Make the Makefile.local
+echo "
+ifeq ($(origin CXX),default)
+FC = g++
```

Comment:
```suggestion
FC = c++
```
but I'm very confused by this at all, claiming that the fortran compiler is `g++` sounds very fishy

Reply:
This might be a typo... I'm copying from `Makefile.local.template` in the main repo